mule-bpm-module/README.md

```diff
@@ -104,6 +112,40 @@ Process factory allows adding variables to process, by using nested `<bpm:proces
 
 By setting `uniqueBusinessKey` attribute, the instance is guaranteed to be the only instance carrying the key.
 
+
+### Process querying & deletion
+
+Processes can be queried and deleted from Java.
```

Comment:
Add example of XML API usage

Reply:
https://github.com/alfameCom/mule-bpm-module/commit/c9bacbfba9158c75e6e5a9e02b99ea043671a895